app/src/components/common/TmSessionImportName.vue

```diff
@@ -6,14 +6,23 @@
       </h2>
       <div class="session-main bottom-indent">
         <Steps :steps="[`Recover`, `Name`, `Password`]" active-step="Name" />
-        <TmFormGroup field-id="import-name" field-label="Your Address">
+        <TmFormGroup field-id="import-name" :field-label="fieldLabel">
```

Comment:
Yes, I need to probably find a better solution, but the TmFormGroup classes were messing `position: absolute` up

Reply:
Fixed